spec.md

```diff
@@ -498,10 +531,6 @@ message VolumeInfo {
   // NFS share). If set, it MUST be non-zero.
   uint64 capacity_bytes = 1;
 
```

Comment:
@saad-ali, thanks for the feedback. As @jdef pointed out, if the CO requested certain `AccessMode` in the request, it probably shouldn't care about what is the *actual* capability of the volume (which might have a larger set of capabilities). For instance, to map to k8s concepts. If a user specified a PVC with access mode `SINGLE_NODE_READER_ONLY`, the PV dynamically provisioned PV for this PVC should probably have access mode `SINGLE_NODE_READER_ONLY` because that's what the user expects the volume to have. Whether the PV actually has `MULTI_NODE_READER_ONLY` capability or not doesn't matter to the user.

Also, I think @jsafrane made an excellent point in #61 that `access_mode` is usually tied to filesystem type. `access_mode` might be different depending on the underlying filesystem. So, returning just `access_mode` probably does not make sense.

I'd be OK that `CreateVolumeResponse` also returns a set of `VolumeCapabilities` that a volume actually has. But as @jsafrane pointed in #61, there might be a lot of combinations. That's the reason I remove those in the spec and always ask the CO to initiate the check using `ValidateVolumeCapabilities` based on what capabilities the user actually want the volume to have.